concourse/pipelines/artifact-releaser-test.jsonnet

```diff
@@ -0,0 +1,616 @@
+// Imports.
+local arle = import '../templates/arle.libsonnet';
+local common = import '../templates/common.libsonnet';
+
+// Get the repository given the build.
+local get_repo(build) = if std.startsWith(build, 'deb') then 'guest-arle-autopush-trusty'
+else if std.startsWith(build, 'el') then 'guest-arle-autopush-el7-x86_64'
+else 'guest-arle-autopush';
+
+// Get the universe given the build.
+local get_universe(build) = if std.startsWith(build, 'deb') then 'cloud-apt'
+else if std.startsWith(build, 'el') then 'cloud-yum'
+else 'cloud-yuck';
+
+// Change '-' to '_', mainly used for images
+local underscore(input) = std.strReplace(input, '-', '_');
+
+local upload_arle_autopush_task = {
```

Comment:
I wonder if we can't reuse whatever is defined within `arle.libsonnet` both here and for the `_stagging` variant, or even maybe have their definition moved to a common place such as `arle.libsonnet` or something.

Reply:
The defined tasks in `arle.libsonnet` can't really be used for packages as far as I understand things; they seem more catered to images. However, the package upload task could definitely be moved to a more common 'library' so-to-speak.